app/components/emails/works/searchworks_component.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Emails
+  module Works
+    # Component for displaying searchworks information for a work.
+    class SearchworksComponent < ApplicationComponent
+    end
+  end
+end
```

Comment:
Can you share why we'd reach for a view component when there's no logic in it vs. making its HTML above a plain ol' Rails partial?

Reply:
Consistency. That was as a developer, you know to look for things in components instead of components AND partials. (In the past, I've done exactly that and ended up confused.)

Also, easier to test. And if it does become more complicated, changes are easier.

That being said, not a very strong preference.

Reply:
I buy that. And components are cheap. Much prefer their specs to view specs.